.github/workflows/support-workers-build.yml

```diff
@@ -96,8 +99,7 @@ jobs:
                     - "-FailureHandlerLambda-"
                     - "-SendAcquisitionEventLambda-"
                     - "-PreparePaymentMethodForReuseLambda-"
-                  fileName: support-workers.jar
-                dependencies: [cfn]
```

Comment:
now the Jar is uploaded first and cfn second, so the old cyclical dependency is gone 🌟